src/function/algebra/decomposition/qr.js

```diff
@@ -53,7 +53,7 @@ function factory (type, config, load, typed) {
    *
    * See also:
    *
-   *    lu
+   *    lusolve
```

Comment:
mathjs has an `lu` decomposition function. We should ideally link to this rather than to `lusolve`. I dunno exactly how to do this though

Reply:
@stropitek the function is called `lup`. I would suggested 

```js
 * See also:
    *
    *    lup, lusolve
```

Reply:
Thanks, I'm going with your suggestion